starters/qwik-fetch-tailwind/src/root.tsx

```diff
@@ -0,0 +1,26 @@
+import { component$ } from '@builder.io/qwik';
+import { QwikCity, RouterOutlet, ServiceWorkerRegister } from '@builder.io/qwik-city';
+import { RouterHead } from './components/router-head/router-head';
+
+import './global.css';
+
+export default component$(() => {
+  /**
+   * The root of a QwikCity site always start with the <QwikCity> component,
+   * immediately followed by the document's <head> and <body>.
+   *
+   * Dont remove the `<head>` and `<body>` elements.
+   */
+  return (
+    <QwikCity>
```

Comment:
Just a generic question, should we use `QwikCity` in this starter project? Or just use the pure `Qwik core` and put `QwikCity` into another `QwikCity` starter repo?